examples/step-49/doc/intro.dox

```diff
@@ -263,7 +263,8 @@ The other four surfaces are declared in a similar way.
 
 Next we define the physical surface itself.  This line is what makes our mesh 2D.
 The values in the braces on the right hand side are the tags for the `Plane Surface`s
-we declared above.
+we declared above.  The number in the parantheses of physical surfaces will become
+the material ID of the mesh as well.
```

Comment:
Add a link to the glossary here as well. This is how you do that:
https://github.com/dealii/dealii/blob/3080f0561f28304c1cf873c6ade1d7980c5e18c5/doc/doxygen/headers/glossary.h#L472

Maybe something like "For more information about material IDs, check the glossary entry..."